src/sdk/namespace_fs.js

```diff
@@ -848,6 +851,19 @@ class NamespaceFS {
                 }
             };
 
+            let previous_key;
+            /**
+             * delete markers are always in the .versions folder, so we need to have special case to determine
+             * if they are delete markers. since the result list is ordered by latest entries first, the first
+             * entry of every key is the latest
+             * @param {Object} obj_info
+             */
+            const set_latest_delete_marker = obj_info => {
```

Comment:
@nadavMiz We need to figure out how we can get isLatest info if we move to the new structure or in an unordered list...

Reply:
@naveenpaul1 on your PR do you already support list object versions unordered as well?

Reply:
@naveenpaul1 maybe in the future